modules/upload_contacts_from_file.py

```diff
@@ -19,7 +19,7 @@ def csv_upload(filepath):
 
 def make_contact_dict(row):
     new_dict = {}
-    new_dict["name"] = row["Name"]
+    new_dict["name"] = u"%s" % (row["Name"])
```

Comment:
I'm curious if the initial `u"%s"` part is necessary or not. (I genuinely don't know.)